commands/json.arrappend.md

```diff
@@ -0,0 +1,52 @@
+Append one or more values to the array values at the path.
+
+## Syntax
+
+```bash
+JSON.ARRAPPEND <key> <path> <json> [json ...]
```

Comment:
Syntax is generated elsewhere right? We shouldn't have it explicitly here.

Reply:
Yeah, let's clean this up. It will be retrieved from the command definition files present in the modules repo. @Nikhil-Manglore